### DISCUSSION TITLE
Implement a hot corner/activity view that shows all windows

### DISCUSSION THREAD

**Discussion by kdgnlx** [lxqt/lxqt#2150] [Ideas]
It's a missing feature and very important one for some users.

Something like this.
https://www.youtube.com/watch?v=MHnIoSq-XhQ

**Comment by tsujan** [comment]
That's the job of WM, not LXQt. KWin/KWinFT, compiz-reloaded and some other WMs have it.

LXQt+ KWin (the hot corner is at the right bottom):

![hot_corner](https://user-images.githubusercontent.com/981076/145849087-7f592059-144c-43a4-98d2-da4816d16be8.jpg)

**Reply by kdgnlx** [reply]
I tried kwin + lxqt, but It runs slower on my old pc. 
I was thinking if it's possible to add it outside the WM.

**Reply by tsujan** [reply]
> I was thinking if it's possible to add it outside the WM.

I'm afraid not.

**Comment by BeatLink** [comment]
Which window manager does LXQT use by default? Perhaps an issue could be filed there?

**Reply by tsujan** [reply]
LXQt is agnostic toward WM and compositor; there's no default. → https://lxqt-project.org/about/
